python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -314,6 +337,38 @@ class LogisticRegression(Base):
         """
         return self.qn.predict(X, convert_dtype=convert_dtype)
 
+    def predict_proba(self, X, convert_dtype=False):
+        """
+        Predicts the class probabilities for X
+
+        Parameters
+        ----------
+        X : array-like (device or host) shape = (n_samples, n_features)
+            Dense matrix (floats or doubles) of shape (n_samples, n_features).
+            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+            ndarray, cuda array interface compliant array like CuPy
+
+        convert_dtype : bool, optional (default = False)
+            When set to True, the predict method will, when necessary, convert
+            the input to the data type which was used to train the model. This
+            will increase memory used for the method.
+        Returns
+        ----------
+        y: cuDF DataFrame
+           Dense matrix (floats or doubles) of shape (n_samples, C)
```

Comment:
Oh, I see, this is really returning a dataframe... I can't wait to get the Array changes in so we can make this more consistent.

Reply:
I would refer to its shape as (n_samples, n_classes)

Reply:
Logged as issue #1711